algorithms.js

```diff
@@ -22,9 +22,9 @@ const binarySearch = (array, item) => {
         if (array[middle] === item) {
             return middle;
         } else if (array[middle] < item) {
-            start = middle +1;
+            start = middle + 1;
```

Comment:
Good :)

Reply:
thx

Reply:
that's all i can fix without your further comments...